src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormulaIT.kt

```diff
@@ -47,174 +38,77 @@ class DataCenterFormulaIT {
 
     @Test
     fun shouldProvisionDataCenter() {
-        val executor = Executors.newFixedThreadPool(
-            2,
-            ThreadFactoryBuilder()
-                .setNameFormat("DCFIT-test-thread-%d")
-                .build()
+        val dataset = DatasetCatalogue().largeJiraEight()
+        val jiraVersion = "8.22.0"
+        val testWorkspace = workspace.isolateTest("shouldProvisionDataCenter")
+        val nonce = UUID.randomUUID().toString()
+        val lifespan = Duration.ofMinutes(30)
+        val keyFormula = SshKeyFormula(
+            ec2 = aws.ec2,
+            workingDirectory = testWorkspace.directory,
+            lifespan = lifespan,
+            prefix = nonce
         )
-        listOf(
-            DataCenterProvisioningTest(
-                jiraVersion = jiraVersionSeven,
-                dataset = datasetSeven
+        val nodeConfigs = JiraNodeConfig.Builder(stableJdk)
+            .remoteJmx(EnabledRemoteJmx())
+            .build()
+            .multipleNodes(2)
+        val dcFormula = DataCenterFormula.Builder(
+            productDistribution = PublicJiraSoftwareDistribution(jiraVersion),
+            jiraHomeSource = dataset.jiraHomeSource,
+            database = dataset.database
+        ).computer(C5NineExtraLargeEphemeral())
+            .databaseComputer(C5NineExtraLargeEphemeral())
+            .waitForRunning(true)
+            .configs(nodeConfigs)
+            .build()
+
+        val provisionedJira = dcFormula.provision(
+            investment = Investment(
+                useCase = "Test Data Center provisioning",
+                lifespan = lifespan
             ),
-            DataCenterJmxProvisioningTest(
```

Comment:
The `shouldProvisionDataCenter` used to spawn 2 threads, one for testing LB, one for testing JMX. I rolled it up into a single test.